apps/fluent-tester/src/FluentTester/theme/applyTheme.ts

```diff
@@ -50,15 +50,14 @@ function applyCaterpillarTheme(parent: Theme): PartialTheme {
 }
 
 const themingModule = getThemingModule()[0];
-const officeBase = themingModule
-  ? createPartialOfficeTheme(themingModule, 'WhiteColors', themingModule.getPalette('WhiteColors') as OfficePalette)
-  : {};
 
 /** apply the currently active theme layering */
 export function applyTheme(parent: Theme, name: ThemeNames): PartialTheme {
   switch (name) {
     case 'Office':
-      return officeBase;
+      return themingModule
+      ? createOfficeTheme({ paletteName: 'WhiteColors' }).theme
+      : {};
```

Comment:
Why is this no longer a partial theme?

Reply:
The partial was just running the base createPartialOfficeTheme part of making a win32 theme. Since I added to the theme recipes, the theme being spit out here wasn't reflecting the "true" result you would get from accessing the theme on win32, so I thought it'd be a good idea to replace this, since we use the tester to show what result you'd get if you just ran the defaults.

Reply:
If the default theme is the new default (lol wording), is there any use of the Office theme? Or is it just an example theme that is only present in our test app?

Reply:
The Office theme is used for win32 still. The default theme is the base, but the Office theme allows for integration with palettes, which can be useful for people on the win32 platform.